### PR TITLE
Hindi to Latin transliteration

### PR DESCRIPTION
**Pull request by bilashsaha**
- `alalc-hin-Deva-Latn-2020.yaml` added

### CODE REVIEW

**Comment by ronaldtse** [comment]
@bilashsaha seems that the tests are failing:

```
1) Interscript maps/alalc-hin-Deva-Latn-2020.yaml system test for {"source"=>"कर्", "expected"=>"rka"}
Failure/Error: raise Interscript::InvalidSystemError.new("No system mappings found")

Interscript::InvalidSystemError:
  No system mappings found
# ./lib/interscript/mapping.rb:85:in `rescue in load_fs_mappings'
# ./lib/interscript/mapping.rb:82:in `load_fs_mappings'
```

You have to remove this line:
```
  inherit: "un-hin-Hind-Latn-2020"
```

File: maps/alalc-hin-Deva-Latn-2020.yaml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    'ह': 'ha'
+
+    # Anusvāra
+    'ं': 'ṃ'
+
+    # Bisarga
+    'ः ': 'ḥ'
+
+    # Anunāsika
+    'ँ': 'n̐' # ঁ : n̐, m̐
```
Comment (ronaldtse) [inline maps/alalc-hin-Deva-Latn-2020.yaml:144]: Can we fully implement this rule?

>   - Anunāsika before guttural, palatal, cerebral, and dental occlusives is transliterated n̐. Before labials, sibilants, semivowels, aspirates, vowels, and in final position it is transliterated m̐.

File: maps/alalc-hin-Deva-Latn-2020.yaml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    # Abagraha
+    'ऽ': '’' # (apostrophe)
```
Comment (ronaldtse) [inline maps/alalc-hin-Deva-Latn-2020.yaml:147]: Does this line already implement this? Is there a special symbol for "double avagraha"?
> When doubled, avagraha is transliterated by two apostrophes

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: maps/alalc-hin-Deva-Latn-2020.yaml (hunk 2 above)
Comment (bilashsaha) [reply maps/alalc-hin-Deva-Latn-2020.yaml:147]: Yes. Double avagraha is just `ऽऽ`.

File: maps/alalc-hin-Deva-Latn-2020.yaml (hunk 1 above)
Comment (bilashsaha) [reply maps/alalc-hin-Deva-Latn-2020.yaml:144]: The use of Anunāsika is very rare in the final position hindi language and its mostly used at the pronunciation as "n". The only word I can remember is `ॐ` which is pronounced as "om" and its a religious word. But yes if we want perfection we can implement this rule.

File: maps/alalc-hin-Deva-Latn-2020.yaml (hunk 2 above)
Comment (ronaldtse) [reply maps/alalc-hin-Deva-Latn-2020.yaml:147]: I see, then can we add an example to test that? Thanks!

File: maps/alalc-hin-Deva-Latn-2020.yaml (hunk 1 above)
Comment (ronaldtse) [reply maps/alalc-hin-Deva-Latn-2020.yaml:144]: Yes please! We'd like to implement it as accurate as possible, and the "om" example will also be good to test against.

Comment (bilashsaha) [reply maps/alalc-hin-Deva-Latn-2020.yaml:144]: Actually at unicode Devanagari "om" is treated as a separate symbol with `U+0950`, So I had used `माँ` as an example here. Also this same rule applies for bengali as well.

Comment (ronaldtse) [reply maps/alalc-hin-Deva-Latn-2020.yaml:144]: Thank you @bilashsaha !

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thank you @bilashsaha ! Nicely done 👍

**Pull request merged**